src/main/scala/com/databricks/spark/xml/XmlReader.scala

```diff
@@ -116,4 +115,11 @@ class XmlReader extends Serializable {
       schema)(sqlContext)
     sqlContext.baseRelationToDataFrame(relation)
   }
+
+  def xmlFile(spark: SparkSession, path: String): DataFrame =
```

Comment:
I was actually thinking about deprecating these too as well. but I think we can do later.